src/main/java/de/tum/in/www1/artemis/web/rest/StudentExamResource.java

```diff
@@ -239,6 +239,11 @@ public StudentExamResource(ExamAccessService examAccessService, StudentExamServi
             return courseAndExamAccessFailure.get();
         }
 
+        // students can not fetch the exam until 5 minutes before the exam start
+        if (ZonedDateTime.now().plusMinutes(5).isBefore(studentExam.getExam().getStartDate())) {
```

Comment:
Can we use a constant for this? There should be a similar constant for the scenario that students get access to the git repos 5 min before the exam starts. It would be great to use the same constant

Reply:
Also make sure to add comments with links to the client constant and vice versa

Reply:
I added the comments and the constant.